crypto/evp/evp_cnf.c

```diff
@@ -44,14 +44,11 @@ static int alg_module_init(CONF_IMODULE *md, const CONF *cnf)
             }
             /*
              * fips_mode is deprecated and should not be used in new
-             * configurations.  Old configurations are likely to ONLY
-             * have this, so we assume that no default properties have
-             * been set before this.
+             * configurations.
              */
-            if (m > 0)
-                EVP_set_default_properties(NULL, "fips=yes");
+            EVP_default_properties_enable_fips(cnf->libctx, m > 0);
         } else if (strcmp(oval->name, "default_properties") == 0) {
-            EVP_set_default_properties(NULL, oval->value);
+            EVP_set_default_properties(cnf->libctx, oval->value);
```

Comment:
Should we be handling the return value (here and above)?

Reply:
Ok changed..